test/e2e/functional_test.go

```diff
@@ -475,7 +475,6 @@ func (s *FunctionalSuite) TestGlobalScope() {
 }
 
 func (s *FunctionalSuite) TestStopBehavior() {
-	s.T().Skip("flaky - see https://github.com/argoproj/argo/issues/2833")
```

Comment:
Test seems to always pass locally. Opening PR to see if I can get a fail in the CI.